src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -263,6 +263,11 @@ public enum DarknessAblaze implements LogicCardInfo {
     return tcgwars.logic.card.Collection.DARKNESS_ABLAZE;
   }
 
+  @Override
+  public int getCollectionLineNo() {
+    return collectionLineNo;
+  }
+
```

Comment:
This is causing build to fail. Was removed in axpendix rework of collection numbers iirc

```suggestion
```